Modules/IO/ImageBase/include/itkImageSeriesReader.hxx

```diff
@@ -479,6 +476,12 @@ ImageSeriesReader<TOutputImage>::GenerateData()
 
   } // end per slice loop
 
+  if (maxSpacingDeviation > 1e-6) // arbitrary number
```

Comment:
This should probably be proportional to the spacing, e.g. if `if (maxSpacingDeviation > 1e-6 * outputSpacing[this->m_NumberOfDimensionsInImage])`

Reply:
@dzenanz I don't really understand what you are suggesting here.

The psuedo code you suggest does not seem valid to me.  Do you mean the product of all spacing sizes times 1e-6?

Reply:
`outputSpacing[this->m_NumberOfDimensionsInImage]` would mostly mean `zSpacing` (a common case with a series of 2D DICOM slices).

Reply:
you are reviewing previous version , by the way